static/js/components/LiveStream.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+/* global SETTINGS: false */
+import React from "react"
+import R from "ramda"
+import { connect } from "react-redux"
+
+import Card from "./Card"
+
+import { actions } from "../actions"
+import { livestreamEventURL } from "../lib/embed"
+
+import type { LiveStreamEvent } from "../flow/livestreamTypes"
+
+type Props = {
+  getLivestream: Function,
+  liveEvent: ?LiveStreamEvent
+}
+
+export class LiveStream extends React.Component<Props> {
+  componentDidMount() {
+    const { getLivestream } = this.props
+
+    if (SETTINGS.livestream_ui_enabled) {
+      getLivestream()
+    }
+  }
+
+  render() {
+    const { liveEvent } = this.props
+
+    if (liveEvent && SETTINGS.livestream_ui_enabled) {
+      const { ownerAccountId, id } = liveEvent
+
+      return (
+        <Card className="widget">
+          <div className="title-row">
+            <span className="title">Live Now @MIT</span>
```

Comment:
Is the title something we'll want to parametrize?

Reply:
for now I don't think so, if we want to add that later it's not so complicated